src/Hl7.Fhir.Base/Rest/BaseFhirClient.cs

```diff
@@ -460,28 +433,44 @@ public virtual async Task DeleteAsync(Resource resource, CancellationToken? ct =
             await DeleteAsync(resource.ResourceIdentity(Endpoint).WithoutVersion(), ct).ConfigureAwait(false);
         }
 
-        /// <summary>
-        /// Delete a resource
-        /// </summary>
-        /// <param name="resource">The resource to delete</param>
-        [Obsolete("Synchronous use of the FhirClient is strongly discouraged, use the asynchronous call instead.")]
-        public virtual void Delete(Resource resource)
-        {
-            DeleteAsync(resource).WaitNoResult();
-        }
-
         /// <summary>
         /// Conditionally delete a resource
         /// </summary>
         /// <param name="resourceType">The type of resource to delete</param>
         /// <param name="condition">Criteria to use to match the resource to delete.</param>
         /// <param name="ct"></param>
+        [Obsolete("As of R6, conditional deletes will be considered separate operations. Use ConditionalDeleteSingle or ConditionalDeleteMultiple instead.")]
```

Comment:
Maybe not mention R6 here yet, but just mention this:
`Use ConditionalDeleteSingle or ConditionalDeleteMultiple instead.`